frontend/src/pages/article/edit-article-page.tsx

```diff
@@ -0,0 +1,21 @@
+import { Layout } from '~/libs/components/components.js';
+import { useLocation } from '~/libs/hooks/hooks.js';
+import { type ArticleWithAuthorType } from '~/packages/articles/articles.js';
+
+import { ArticleForm } from './components/components.js';
+import styles from './styles.module.scss';
+
+const EditArticlePage: React.FC = () => {
+  const location = useLocation();
+  return (
+    <Layout>
+      <div className={styles.articleCreatePageWrapper}>
+        <ArticleForm
+          articleForUpdate={location.state as ArticleWithAuthorType}
```

Comment:
Maybe let's check if location.state is not empty before passing it

Reply:
And what I should do if it's empty ? return empty div ?

Reply:
Redirect back to my-articles